lib/middleware.ts

```diff
@@ -86,6 +103,14 @@ var defaultHandlers = [
   (req, res) => ({"status-code": res.statusCode}),
   // Ip address
   (req) => ({ip: getIp(req)}),
+
+  // Kayvee's reserved fields
+  // Log level
+  (req, res) => ({level: getLogLevel(req, res)}),
+  // Source
+  () => ({source: "kayvee"}),
```

Comment:
Prevent user from modifying `morgan_options` .. 2e76a05

It now only gets modified during tests in order to allow logging to a stream (for comparing output)